tests/integration/targets/aci_vrf_fallback_route_group/tasks/main.yml

```
@@ -0,0 +1,411 @@
+        fallback_route: 1.1.1.1/2
+        fallback_members: 
+          - 1.1.1.1
+          - 2.2.2.2
+          - 4.4.4.4
+        description: Test for Fallback Route Group
+        state: present
+      check_mode: true
+      register: cm_create_vrf_fbrg
+
+    - name: Create vrf fallback route - Normal mode
+      cisco.aci.aci_vrf_fallback_route_group:
+        <<: *fbrg_info
+      register: nm_create_vrf_fbrg
+
+    - name: Sort children for fallback group - Check mode
+      ansible.builtin.set_fact:
+        cm_sorted_children: "{{ cm_create_vrf_fbrg.proposed.fvFBRGroup.children | map('dict2items') | sort(attribute='0.key') | map('items2dict') | list }}"
+    
+    - name: Sort fallback members for fallback group - Check mode
+      ansible.builtin.set_fact:
+        cm_sorted_child_members: "{{ cm_create_vrf_fbrg.proposed.fvFBRGroup.children | selectattr('fvFBRMember', 'defined') | map(attribute='fvFBRMember.attributes') | list | sort(attribute='rnhAddr') }}"
+
+    - name: Sort children for fallback group
+      ansible.builtin.set_fact:
+        sorted_children: "{{ nm_create_vrf_fbrg.current[0].fvFBRGroup.children | map('dict2items') | sort(attribute='0.key') | map('items2dict') | list }}"
+    
+    - name: Sort fallback members for fallback group
+      ansible.builtin.set_fact:
+        sorted_child_members: "{{ nm_create_vrf_fbrg.current[0].fvFBRGroup.children | selectattr('fvFBRMember', 'defined') | map(attribute='fvFBRMember.attributes') | list | sort(attribute='rnhAddr') }}"
+
+    - name: Verify that vrf_fbrg has been created correctly with the correct attribute values
+      ansible.builtin.assert:
+        that:
+          - cm_create_vrf_fbrg is changed
+          - cm_create_vrf_fbrg.previous == []
+          - cm_create_vrf_fbrg.proposed.fvFBRGroup.attributes.name == "test_fallback_route_group"
+          - cm_sorted_child_members.0.rnhAddr == "1.1.1.1"
+          - cm_sorted_child_members.1.rnhAddr == "2.2.2.2"
+          - cm_sorted_child_members.2.rnhAddr == "4.4.4.4"
+          - cm_sorted_children.3.fvFBRoute.attributes.fbrPrefix == "1.1.1.1/2"
+          - nm_create_vrf_fbrg is changed
+          - nm_create_vrf_fbrg.previous == []
+          - nm_create_vrf_fbrg.current.0.fvFBRGroup.attributes.name == "test_fallback_route_group"
+          - sorted_children.3.fvFBRoute.attributes.fbrPrefix == "1.1.1.1/2"
+          - sorted_child_members.0.rnhAddr == "1.1.1.1"
+          - sorted_child_members.1.rnhAddr == "2.2.2.2"
+          - sorted_child_members.2.rnhAddr == "4.4.4.4"
+
+    - name: Create vrf fallback route again (idempotency)
+      cisco.aci.aci_vrf_fallback_route_group:
+        <<: *fbrg_info
+      register: nm_create_vrf_fbrg_again
+    
+    - name: Verify that nm_create_vrf_fbrg_again is not changed (idempotency)
+      ansible.builtin.assert:
+        that:
+          - nm_create_vrf_fbrg_again is not changed
+          - nm_create_vrf_fbrg_again.current == nm_create_vrf_fbrg.current
```

why are we not checking previous here?
```suggestion
          - nm_create_vrf_fbrg_again .previous == nm_create_vrf_fbrg_again.current == nm_create_vrf_fbrg.current
```